Assets/_BForBoss/_Core/Scripts/Environment/EnemySpawnAreaBehaviour.cs

```diff
@@ -12,6 +17,8 @@ public class EnemySpawnAreaBehaviour : MonoBehaviour
         [SerializeField] [Min(0.0f)] private float _secondsBetweenSpawns = 2.5f;
         [SerializeField, Tooltip("Should max amount of enemies spawn upon initialization")]
         private bool _burstInitialSpawn = true;
+        [SerializeField] 
+        private float _spawnArea = 1;
```

Comment:
What is this supposed to be? A multiplier for the positioning?

Does this have to be exposed? If so, a better naming needs to be used

Reply:
yeah i just updated it. However i'm thinking of a better way to phrase this since it's a bit of a lie. 

1. it first get's a random spot within a radius
2. It then get's closest navmesh point. HOWEVER, the closest navmesh point might not be within that specific radius. 

Sooooooooo, i'm trying to think of something a bit more intuitive, or just not expose it at all.